Makefile

```
@@ -0,0 +1,71 @@
+# .DEFAULT_GOAL := all
+sources = src test
+
+.PHONY: .uv  ## Check that uv is installed
+.uv:
+	@uv -V || echo 'Please install uv: https://docs.astral.sh/uv/getting-started/installation/'
+
+.PHONY: install  ## Install the package, dependencies, and pre-commit for local development
```

```suggestion
.PHONY: install  ## Install the package and dependencies for local development
```
We will add pre-commit later :)